### DISCUSSION TITLE
main screen detection stay on top

### DISCUSSION THREAD

**Discussion by kiloptero** [blakeblackshear/frigate#12486] [General]
![image](https://github.com/user-attachments/assets/30591fe2-16fe-491c-8d06-0eff3e456312)

hi! there is a way to mantain the detection videos in the top of the main screen? a few hours or minute after detection they disappear. its no so important but will be great if there is a config to stay there until i see.

![image](https://github.com/user-attachments/assets/9a064eb6-f906-44dd-925b-10925ee70e34)

**Comment by hawkeye217** [comment]
The area above your cameras in Live view is a display of the most recent activity (alerts) on your cameras. They will remain in that area for 2 hours or until you:

- review them by clicking/tapping on them 
- or review them from the Review page

This can't be changed at the moment. However, all of your unreviewed items are visible on the Review page, and reviewed items are also a single tap/click away by clicking the "Show reviewed" switch/icon.

**Comment by kiloptero** [comment]
thanks:)

Andres Roepke Del Solar.
Ingeniero Civil Industrial
Ingeniero Electronico
Mobile 88196932
